yaramo/signal.py

```diff
@@ -79,3 +80,23 @@ def distance_previous_node(self):
             return self.distance_edge
         else:
             return self.edge.length - self.distance_edge
+
+    def to_serializable(self) -> Tuple[dict, dict]:
+        attributes, _ = super().to_serializable()
+        references = {
+            'edge': self.edge.uuid if self.edge else None,
+            'trip':self.trip.uuid if self.trip else None,
+            'additional_signals': [signal.uuid for signal in self.additional_signals],
+            'direction': str(self.direction),
+            'side_distance': self.side_distance,
+            'function': str(self.function),
+            'kind': str(self.kind),
+        }
+        objects = {}
+        items = [self.trip] + self.additional_signals if self.trip else self.additional_signals
```

Comment:
The if and else cases are identical here, so could this just be `items = [self.trip] + self.additional_signals`?

Reply:
No, I don't want a `None` in my `items` array. Would you prefer a filter of existing items?